content/en/blog/how-to-use-fission-functions-with-postgresql-database/index.md

```diff
@@ -0,0 +1,357 @@
++++
+title = "How to use Fission functions with PostgreSQL database"
+date = "2021-12-27T12:28:34+05:30"
+author = "Atulpriya Sharma"
+description = "Tutorial on how to use Fission functions with PostgreSQL database on Kubernetes Cluster."
+categories = ["Tutorial","Application","Python","Database"]
+type = "blog"
++++
+
+Software development and deployment methods have evolved greatly over the years.
+From the times when system admins used to setup servers to run software applications to today when developers are not concerned about how the server is provisioned. Containers and container orchestration tools like Kubernetes help developers deploy serverless applications quickly.
+Similarly, serverless functions like [Fission](https://fission.io) help you focus on the code rather than the infrastructure.
+
+In this blog post, I'll show you how to use **Fission functions with PostgreSQL database**.
```

Comment:
We will use PostgresSQL to persist data of operations performed by the user.

Reply:
updated